mlir/lib/Ion/Transforms/oqd_database_managers.h

```diff
@@ -0,0 +1,36 @@
+#include <dlfcn.h>
+#include <iostream>
+
+//#include <nanobind/nanobind.h>
+//#include <nanobind/eval.h>
+//#include <nanobind/stl/string.h>
+
+using namespace std;
+
+void oqd_hi(){
```

Comment:
I will convert this to hpp. The reason I did not make this a cpp is that since this is the mlir directory, all cpp files must be targets (demanded by llvm), and I don't want this extra layer of complexity for this "header".